docs/sites.yml

```diff
@@ -5303,3 +5303,14 @@
   built_by: Ste O'Neill
   built_by_url: https://steoneill.dev
   featured: false
+- title: Filipe Santos Correa's Portfolio
+  description: >
+    Filipe's Personal About Me / Portfolio.
+  main_url: "https://filipesantoscorrea.com/"
+  url: "https://filipesantoscorrea.com/"
+  source_url: "https://github.com/Safi1012/filipesantoscorrea.com"
+  featured: false
+  categories:
+    - About Me
```

Comment:
We don't have this category and I think "Personal Website" category encapsulate this so would prefer to remove this one:

Reply:
Thank you for you suggestion! I just updated it ;)